documentation/EnableConfiguration.md

```diff
@@ -1,8 +1,15 @@
 ﻿# Enabling **stylecop.json**
 
-At this time, the code fix is not able to fully configure the newly-created **stylecop.json** file for use. This is
-tracked in bug report [dotnet/roslyn#4655](https://github.com/dotnet/roslyn/issues/4655). In the mean time, users must
-manually perform the following additional steps after creating the **stylecop.json** file.
+## StyleCop.Analyzers v1.2 and above
+
+If you are using version 1.1.1 or newer, you no longer need to include stylecop.json using the methods below.  It is
+automatically included if it is next to the project or in a directory above the project.  
+
+## StyleCop.Analyzers v1.1 and below
```

Comment:
:bulb: Technically there is not yet a v1.1 stable package, so we don't need to increment the version. If you want, you can be specific in the text above to say:

> If you are using version 1.1.0 RC 2 or newer and place your **stylecop.json** file in the project directory or one of its ancestors (often including the solution directory), ...